recipe/build.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 
-python ${RECIPE_DIR}/brand_python.py
+${SYS_PYTHON} ${RECIPE_DIR}/brand_python.py
+
+# Remove test data and ensurepip stubs to save space
+rm -rf Lib/test Lib/*/test
+rm -rf Lib/ensurepip
```

Comment:
Removing these files saves a significant amount of space, perhaps the same should be done in the Window build.